Automattic/wp-calypso
# e2e: update chromedriver for local use

#### Changes proposed in this Pull Request

* This updates the package of chromedriver we are using. This is primarily for local use since CI reads from the .chromedriver file. That will be updated when the CircleCI image updates Chrom

#### Testing instructions
* Try to run e2e tests locally and make sure you can run some of them.
* Ensure CircleCI e2e tests pass.
